bigquery/docker-compose.yml

```diff
@@ -10,7 +10,8 @@ services:
       # deployments; bootstrap checks are still enabled for production
       # deployments.
       - "transport.host=localhost"
-      - "ES_JAVA_OPTS=-Xms2g -Xmx2g"
+      - "HEAP_SIZE=${HEAP_SIZE:-256m}"
```

Comment:
Add a bullet to README:
```
* Run Elasticsearch:
  * If you intend to run the [Data Explorer UI](https://github.com/DataBiosphere/data-explorer/)
  after this, run inside the *data-explorer* repo:
    ```
    docker-compose up -d elasticsearch
    ```
  * If you do not intend to run the Data Explorer UI after this, and just want
  to inspect the index in Elasticsearch, run inside this repo from the
  `bigquery` directory:
    ```
    docker-compose up -d elasticsearch
    ```
  * If ES crashes due to OOM, you can increase memory:
    ```
    HEAP_SIZE=10g docker-compose up -d elasticsearch
    ```
```

Reply:
Done.